reactor-core/src/main/java/reactor/util/Loggers.java

```diff
@@ -573,17 +625,17 @@ public boolean isInfoEnabled() {
 
 		@Override
 		public synchronized void info(String msg) {
-			this.log.format("[ INFO] (%s) %s\n", Thread.currentThread().getName(), msg);
+			this.log.format("[INFO] (%s) %s\n", Thread.currentThread().getName(), msg);
 		}
 
 		@Override
-		public synchronized void info(String format, Object... arguments) {
-			this.log.format("[ INFO] (%s) %s\n", Thread.currentThread().getName(), format(format, arguments));
+		public void info(String format, Object... arguments) {
+			logWithOptionalThrowable("INFO", format, arguments);
```

Comment:
I am guessing the space was to align INFO under DEBUG and such in a single column so the space was a filler. With the spaces all of (TRACE, DEBUG, INFO, WARN, ERROR) take up the same amount of horizontal space. Please roll back.

Reply:
Thanks for detailed explanation!
I have reverted the code.